### DISCUSSION TITLE
Feedback on Binder Error with Composite UNIQUE/PRIMARY KEY Constraints in UPDATE Operation

### DISCUSSION THREAD

**Discussion by yishaojun** [duckdb/duckdb#12340] [Q&A]
![PixPin_2024-06-01_15-22-14](https://github.com/duckdb/duckdb/assets/84978692/bfeaa4fa-566b-437f-a352-422473971ef6)

Dear DuckDB Development Team,

I am reaching out to provide feedback on an issue I encountered with the DuckDB database system. The problem arises when attempting to perform an UPDATE operation on a table that has multiple unique or primary key constraints.

Issue Description:
When I attempt to UPDATE a table with multiple unique or primary key constraints, DuckDB throws the following error:

Binder Error: Conflict target has to be provided for a DO UPDATE operation when the table has multiple UNIQUE/PRIMARY KEY constraints
The error suggests that I need to provide a clear target for the UPDATE operation, but in my understanding, I have already provided sufficient information to locate specific rows through the WHERE clause.
DuckDB Version: V0.9

**Comment by Tishj** [comment]
The code you have provided is not enough context to know what you're having issues with
You mention a WHERE clause, yet none is provided. You mention having multiple unique or primary key constraints while only showing one.

Please provide this information and do so in text form :+1:
